flutter/analysis_options.yaml

```diff
@@ -11,6 +11,8 @@ analyzer:
     # allow self-reference to deprecated members (we do this because otherwise we have
     # to annotate every member in every test, assert, etc, when we deprecate something)
     deprecated_member_use_from_same_package: warning
+    # ignore sentry/path on pubspec as we change it on deployment
+    invalid_dependency: ignore
```

Comment:
@rxlabz ideas why this works locally but it fails on CI?

> warning • 'invalid_dependency' isn't a recognized error code • analysis_options.yaml:15:5 • unrecognized_error_code

CI runs flutter pub get, so versioning should be the same